jekyll/_cci2/server/installation/phase-2-core-services.adoc

```diff
@@ -500,6 +500,81 @@ workloadIdentity: <workload-identity-service-account-email>
 // Stop hiding from AWS PDF
 endif::env-aws[]
 
+
+== Example Manifests
+The following are example manifests that contain the basic required parameters necessary to spin up the circleci-server helm installation.
+
+=== AWS
+
+```yaml
+
+```
+
+=== GCP
+The below is an example manifest of the necessary parameters for an installation of circleci-server in a GCP environment. Note that this installation uses Workload Identity, which is recommended. Fields with base64 encoding are marked as such.
+
+```yaml
+global:
+  domainName: "<full-domain-name-of-your-install>"
+  license: '<license-for-circleci-server>'
+
+tls:
+  certificate: "<generated-base64-tls-cert-with-certbot>"
+  privateKey: "<generated-base64-tls-key-with-certbot>"
```

Comment:
This can be omitted.  The default self signed certs is a customers preference for getting off the ground quickly: https://circleci.slack.com/archives/G013L90K674/p1656595069663909